pages/_app.tsx

```diff
@@ -29,9 +33,22 @@ const App = React.memo(function App(props: AppProps) {
   }
 
   return (
-    <div className='app'>
-      <Route {...routeProps} />
-    </div>
+    <RenderErrorBoundary onError={onAppRenderError}>
+      {({ renderError }) => (
+        <>
+          {renderError ? (
+            <>
+              <p>An error occurred. If error persists, please contact the Experiments Platform team.</p>
```

Comment:
Not relevant to this PR, but @yanirs, what do you think about getting "experiment-platform@automattic.com":

```suggestion
              <p>An error occurred. If error persists, please contact the <a href='mailto:experiment-platform@automattic.com'>Experiments Platform team</a>.</p>
```

Or we could create a public p2/forum (eventually)...

Reply:
Can leave the email out for now. :smile: 

If we want a link, let's link to the P2: https://betterexperiments.wordpress.com/.

Can add make it a public P2 if and when we go public. :slightly_smiling_face: